src/modules/Dropdown/Dropdown.js

```diff
@@ -460,9 +461,19 @@ export default class Dropdown extends Component {
     // open search dropdown on search query
     if (search && newQuery && !open) this.open()
 
+    // resize the search input, temporarily show the sizer so we can measure it
+    let searchWidth
+    if (newQuery) {
+      const sizer = findDOMNode(this).querySelector('.sizer')
```

Comment:
Well you essentially have a ref by doing `findDOMNode(this)`, so it's just being more direct about the matter.